android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/utils/ErrorFormatter.java

```diff
@@ -0,0 +1,32 @@
+package com.shopify.unity.buy.utils;
+
+import com.google.android.gms.wallet.WalletConstants;
+
+public final class ErrorFormatter {
```

Comment:
Probably better naming is to be `WalletErrorFormatter` as if I'm not wrong this only for android pay wallet errors